pages/_index.html

```diff
@@ -72,7 +72,7 @@ <h3>4</h3>To finish the procedure, make a pull request in our GitHub
 
 <div id="cad" >
     <script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?serve=CESD4K3E&placement=jsorg" id="_carbonads_js"></script>
-    <span id="cad-blocked">Looks like your ad blocker did a great job.<br>Here would have been a paid advertisement that would have helped us to run this service.</span>
+    <span id="cad-blocked">Your AD blocker worked! But to keep this service going we need support, consider <a href="https://opencollective.com/js-org">donating to us</a> :)</span>
```

Comment:
This proposed wording comes across in a very begging way, asking for donations, which I'm not personally a fan of.

I think far more gentle wording could be used:

```suggestion
    <span id="cad-blocked">Looks like your ad blocker worked.<br>This would have been a paid advertisement that helped js.org to continue operating.<br>Please consider supporting us on <a href="https://opencollective.com/js-org">OpenCollective</a>, so that we don't need ads.</span>
```

Reply:
Sure